crates/api_common/src/person.rs

```diff
@@ -243,11 +238,58 @@ pub struct GetPersonDetailsResponse {
   pub person_view: PersonView,
   #[cfg_attr(feature = "full", ts(optional))]
   pub site: Option<Site>,
-  pub comments: Vec<CommentView>,
-  pub posts: Vec<PostView>,
   pub moderates: Vec<CommunityModeratorView>,
 }
 
+#[skip_serializing_none]
+#[derive(Debug, Serialize, Deserialize, Clone, Default, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+/// Gets a person's content (posts and comments)
+///
+/// Either person_id, or username are required.
+pub struct ListPersonContent {
+  #[cfg_attr(feature = "full", ts(optional))]
+  pub person_id: Option<PersonId>,
+  /// Example: dessalines , or dessalines@xyz.tld
+  #[cfg_attr(feature = "full", ts(optional))]
+  pub username: Option<String>,
+  #[cfg_attr(feature = "full", ts(optional))]
+  pub page_cursor: Option<PersonContentCombinedPaginationCursor>,
+  #[cfg_attr(feature = "full", ts(optional))]
+  pub page_back: Option<bool>,
+}
```

Comment:
These will need an option like post_only or comment_only if you want to keep the current ui with tabs for Overview / Posts / Comments etc. Otherwise the PR looks fine.

Reply:
Good call, I'll add that now.

Reply:
K that's done.